.github/workflows/deploy.yml

```diff
@@ -0,0 +1,82 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10.x"
+
+      - name: Pull down dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build:shared && npm run build:browsers && ng build --prod
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-2
+
+      - name: Log in to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Docker Build, tag and push image to Amazon ECR
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: tis-revalidation-v2
```

Comment:
if anyone knows how to lowercase and set an envar please let me know

Reply:
This for re-using the repo name for the ECR repo?

Reply:
If so, I'm happy to rename the repo. Did the same with `tis-trainee-details` for similar reasons.

Reply:
yea, its so that we can create images with lowercase names, struggled with it in the other project so I just hardcoded it for now